### PR TITLE
Add list of add-ins using exchange online legacy tokens

### PR DESCRIPTION
**Pull request by davidchesnut**
To support developer and administrator efforts to identify add-ins that are using legacy Exchange online tokens, this PR provides a spreadsheet listing all Microsoft store add-ins known to still be using legacy tokens as of October 2024. The readme includes steps for how to cross-reference deployed add-ins in your tenant with the spreadsheet.

### CODE REVIEW

**Comment by davidchesnut** [comment]
Please don't merge until I give the okay.

File: add-in-ids/README.md
Comment (AlexJerabek) [inline add-in-ids/README.md]: Grammar suggestion. Also, not sure if "deployed" is redundant.
```suggestion
Microsoft 365 administrators should use this spreadsheet to determine if any deployed Outlook add-ins in your tenant are potentially using legacy Exchange online tokens.
```

File: add-in-ids/README.md
Comment (AlexJerabek) [inline add-in-ids/README.md]:
```suggestion
Use the following steps in PowerShell to get a list of all add-ins deployed on your tenant.
```

File: add-in-ids/README.md
Comment (AlexJerabek) [inline add-in-ids/README.md]: Grammar suggestions.
```suggestion
This will list all add-ins deployed on your tenant. There will be a **DisplayName** column and a **ProductId** column. Look up any add-ins by searching for the **ProductId** in the spreadsheet. If the add-in is in the spreadsheet, then it uses legacy Exchange online tokens (as of October 2024).
```

File: add-in-ids/README.md
Comment (AlexJerabek) [inline add-in-ids/README.md]:
```suggestion
## What to do with add-ins that use legacy tokens
```

File: add-in-ids/README.md
Comment (AlexJerabek) [inline add-in-ids/README.md]:
```suggestion
If you have deployed any add-ins that are listed in the spreadsheet, we recommend that you reach out to those publishers to ensure they migrate to Entra ID tokens. Otherwise, when legacy Exchange tokens are eventually turned off, those add-ins will not work correctly.
```

File: add-in-ids/README.md
Comment (AlexJerabek) [inline add-in-ids/README.md]:
```suggestion
If your tenant doesn’t have any add-ins that use legacy Exchange tokens, we recommend you turn off Exchange tokens in your tenant as a security best practice.
```

**Review by AlexJerabek — DISMISSED** [review]
Approving with some phrasing suggestions.

**Review by lindalu-MSFT — DISMISSED** [review]
Nothing from me, looks like Alex already made the suggested changes!

**Review by lindalu-MSFT — APPROVED** [review]

**Pull request merged**
